diffgraph/structural.py

```
@@ -0,0 +1,408 @@
+    text_hash: str
+
+
+@dataclass(frozen=True)
+class _Import:
+    module: str
+    line: int
+    snippet: str
+
+
+def _run(repository: str, *args: str) -> bytes:
+    return subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)